GPCS4/Emulator/Linker.cpp

```diff
@@ -94,16 +94,18 @@ bool CLinker::resolveSymbol(MemoryMappedModule const &mod,
 
 			if (address == nullptr)
 			{
+				// NOTE: Something is wrong with va_args and u64 values, so print NID as 2 u32
 				formatString =
-					"Unknown Function nid 0x%016x from lib:%s is called: 0x%08x";
+					"Unknown Function nid 0x%08x%08x from lib:%s is called: 0x%08x";
 			}
 			else
 			{
 				formatString =
-					"Function nid 0x%016x from lib:%s is called";
+					"Function nid 0x%08x%08x from lib:%s is called";
```

Comment:
just suggestion, maybe better show base64 name? for easier search on aerolib.scv.
it's not this PR, but for future

Reply:
True, done.